app/assets/stylesheets/application.tailwind.css

```diff
@@ -464,15 +470,15 @@ body {
   @apply text-sm font-normal text-slate-500 dark:text-slate-400;
 }
 
-div.field_with_errors> :is(select) {
+div.field_with_errors > :is(select) {
   @apply text-red-900 placeholder-red-300 border-red-700;
 }
 
-.table-container table:not(:has(td)) {
+.table-container table:not(:has(tbody td)) {
   display: none;
 }
 
-.table-container:not(:has(td)) .empty_state_message {
+.table-container:not(:has(tbody td)) .empty_state_message {
```

Comment:
Why did you increase the specificity here to `tbody td`?  `td` can only exist within a `tbody` so it's kind of redundant.  If there is a case that there is a `td` instead of a `th` in the `theaad` then we need to address that semantic issue there.

Reply:
Had accessibility errors for adding empty <th> in <thead> for checkbox column.

Reply:
Sorry, my comment was wrong here anyways.  Thanks for fixing this.